clowdapp.yaml

```diff
@@ -177,6 +177,37 @@ objects:
               SB.ID as BASELINE_ID,
               SBMS.SYSTEM_ID::TEXT AS ASSOCIATED_SYSTEM_ID FROM SYSTEM_BASELINES SB
               LEFT OUTER JOIN SYSTEM_BASELINE_MAPPED_SYSTEMS SBMS ON SBMS.SYSTEM_BASELINE_ID  = SB.ID;
+  - apiVersion: metrics.console.redhat.com/v1alpha1
+    kind: FloorPlan
+    metadata:
+      name: system-baseline-hms
+    spec:
+      database:
+        secretName: ${FLOORIST_DB_SECRET_NAME}
+      objectStore:
+        secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+      suspend: ${{FLOORIST_SUSPEND}}
+      logLevel: ${FLOORIST_LOGLEVEL}
+      queries:
+        - prefix: hms_analytics/drift/baselines
+          query: >-
+            SELECT
+              SB.ACCOUNT AS ACCOUNT_ID,
+              SB.ID AS BASELINE_ID,
+              SB.ORG_ID,
+              SB.CREATED_ON,
+              SB.MODIFIED_ON,
+              SB.NOTIFICATIONS_ENABLED,
+              JSON_AGG(SBMS.SYSTEM_ID) AS ASSOCIATED_SYSTEM_IDS
```

Comment:
@shaded-enmity do you have experience with having JSON formatted data in a Parquet format? I do not seem to find it in the [supported Parquet types](https://parquet.apache.org/docs/file-format/types/).

Reply:
Here's how it works in Image Builder, https://github.com/osbuild/image-builder/blob/2139543853a11e41577664d36e39422c618bac4b/templates/image-builder.yml#L158
packages + repositories is a list of objects. I think it's handled under the hood via the Logical Types, but I haven't looked at how exactly this is implemented.